csharp/src/SeedLang/Block/Converter.cs

```diff
@@ -96,9 +96,9 @@ public class Converter {
         foreach (var rootBlock in module.RootBlockIterator) {
           // TODO: implement a visitor to parse other kinds of blocks.
           if (rootBlock is ExpressionBlock expressionBlock) {
-            var parser = new InlineTextParser();
-            if (parser.Parse(expressionBlock.GetEditableText(), module.Name, ParseRule.Expression,
-                             collection, out AstNode node, out IReadOnlyList<SyntaxToken> _)) {
+            var parser = new BlockInlineTextParser();
```

Comment:
As for the naming:

* It seems all the functions that convert SeedX source code to syntax tree are named as `ParseXXX` or similar patterns.
* Meanwhile, SeedBlock uses a particular name pattern `Converter.Convert` to do a different but kind of similar task.
* And, the naming in `BlockInlineTextParser.cs` does follow the `ParseXXX` pattern like other SeedX languages.

I am thinking if the name `Convert` is a good fit here, with no preference yet. Just some random thoughts:

* `Convert` is short and clean, but it looks a bit out-of-context compared to other `ParseXXX` names.
* Not sure if `Translate` is a better naming solution...
* Not sure if `Parser...` still fits here...

Anyway, we can keep `Convert` for now and postpone this decision...

Reply:
I don't think Converter is good here too. But let's keep it open now. Maybe we can find a better name after we finalize how to handle the workflow of editing and converting inline texts to blocks.